src/helpers/helper.js

```diff
@@ -0,0 +1,10 @@
+export const prepareObject = (item) => {
+  const objectItem = item.toJSON();
+  const { _id, value, completed, isEditing } = objectItem;
+  return {
+    id: _id,
+    value: value,
+    completed: completed,
+    isEditing: isEditing,
```

Comment:
We use this field only on frontend, and in DB it always will be false. Lets remove it from Todo schema and append only on frontend